skymp5-server/cpp/lib_espm/espm.cpp

```diff
@@ -59,6 +59,14 @@ class RefrKey
 
   const uint64_t v;
 };
+
+std::map<std::string, uint32_t> correctHeshcode = {
+  std::map<std::string, uint32_t>::value_type("Skyrim.esm", 0xaf75991dUL),
```

Comment:
Document the meaning of magic constants. 
For example by using `enum` like here: https://github.com/Dinakrab/skymp/blob/3cba8fed539810e840df39d327e3da646deb9e9f/skymp5-server/cpp/lib_espm/espm.cpp#L114

Reply:
It's already part of a constant actually, I guess there's no need to make a separate constant, since it probably won't be used anywhere else outside of this value

Reply:
I think there is nothing to document, `correctHeshcode` indicates that there are hashes, and "Skyrim.esm" and other string constants indicate what master file is it.